src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void writeToFile(String pathToFile, String report) {
+        Path path = Path.of(pathToFile);
+        try {
+            Files.writeString(path, report, StandardOpenOption.CREATE);
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write file " + path, e);
```

Comment:
The error message in the `RuntimeException` uses a backtick (`) instead of an apostrophe ('). Consider changing `Can`t write file` to `Can't write file` for correct punctuation.